.github/workflows/tests.yml

```diff
@@ -40,8 +44,8 @@ jobs:
 
       - name: Install shiny.benchmark
         run: |
-          R -e "install.packages('remotes')"
-          R -e "remotes::install_github(repo = 'Appsilon/shiny.benchmark', auth_token = Sys.getenv('GITHUB_PAT'), ref = Sys.getenv('BRANCH_NAME'), quiet = TRUE)"
+          Rscript -e "install.packages('remotes')"
+          Rscript -e "remotes::install_local('../../../', quiet = TRUE)"
```

Comment:
The problem here is in the case you are testing a new feature in a specific branch. The tests are going to fail since it will install the develop version of the package. Is it smart enough to install the current branch?

Reply:
The checkout action uses exactly the commit that was the trigger, so this should work fine.